test/Operators/spectralelement/benchmark_kernels.jl

```diff
@@ -145,3 +145,25 @@ function kernel_ntuple_field_dss!(args)
     Spaces.weighted_dss_ghost!(nt_ϕψ, nt_ϕψ_buffer)
     return
 end
+
+##### ntuple_floats dss!
+kernel_ntuple_floats_dss_array!(args) = kernel_copyto!(args)
+function kernel_ntuple_floats_dss!(args)
+    (; nt_ϕψ_ft_buffer) = args.buffers
+    (; nt_ϕψ_ft) = args
+    Spaces.weighted_dss_start!(nt_ϕψ_ft, nt_ϕψ_ft_buffer)
+    Spaces.weighted_dss_internal!(nt_ϕψ_ft, nt_ϕψ_ft_buffer)
+    Spaces.weighted_dss_ghost!(nt_ϕψ_ft, nt_ϕψ_ft_buffer)
+    return
+end
+
+##### complicated_field dss!
+kernel_complicated_field_dss_array!(args) = kernel_copyto!(args)
+function kernel_complicated_field_dss!(args)
+    (; f_comp_buffer) = args.buffers
+    (; f_comp) = args
+    Spaces.weighted_dss_start!(f_comp, f_comp_buffer)
+    Spaces.weighted_dss_internal!(f_comp, f_comp_buffer)
+    Spaces.weighted_dss_ghost!(f_comp, f_comp_buffer)
```

Comment:
We can just do `Spaces.weighted_dss(...)` unless there is a good reason to separate them into three function calls!